### PR TITLE
Game Modifiers - Images and Tables

### PR DESCRIPTION
**Pull request by EpicSupermind**
Moved images out of headers, did not resize. Added Score Multiplier and Visual Settings Tables.
Also kinda messed up and put some hoo in here :/

### CODE REVIEW

**Comment by MegaApplePi** [comment]
please change the title of this pr to something like "Fix images and tables in Game Modifiers" -- `Images and Tables` can basically mean anything really

File: wiki/Game_Modifiers/en.md [hunk 1]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
 
 ***Score multiplier: 0.70 ~ 1.00 (osu!mania only)***
 
 ***"Double the key amount, double the fun!"***
 
-[Introduced in 5th of March, 2015 build](http://osu.ppy.sh/p/changelog?v=b20150305), this mod will introduce a second playfield with second set of control based on current xK count (7K -&gt; 7K + 7K). For osu!mania specific mode, the xK is split to half (7K -&gt; 4K + 3K). It is entirely possible to increase the key count to *18 Keys* (9K \[mod\] -&gt; 9K + 9K, applied to non-specific difficulties only).
+[Introduced in 5th of March, 2015 build](https://osu.ppy.sh/p/changelog?v=b20150305), this mod will introduce a second playfield with second set of control based on current xK count (7K -&gt; 7K + 7K). For osu!mania specific mode, the xK is split to half (7K -&gt; 4K + 3K). It is entirely possible to increase the key count to *18 Keys* (9K \[mod\] -&gt; 9K + 9K, applied to non-specific difficulties only).
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use escaped greater than signs instead of Character Entities

File: wiki/Game_Modifiers/en.md [hunk 2]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
 
 [A detailed explanation about the second playfield can be found in the osu!mania page.](Osu!mania#Co-Op "wikilink")
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: change line to
```
A detailed explanation about the second playfield can be found in the [osu!mania page](/wiki/Game_Modes/osu!mania#co-op).
```

File: wiki/Game_Modifiers/en.md [hunk 3]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
 
 Score multiplier only applies within this condition:
 
 **This table only affects beatmaps converted from Standard to osu!mania during play with Co-Op mod on**.
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use either `osu!` or `osu!standard` but not `Standard`

File: wiki/Game_Modifiers/en.md [hunk 4]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
 
-**The Co-Op mod gives <span style="color:red">NO penalty</span> on osu!mania-specific beatmaps.**
+**The Co-Op mod gives NO penalty on osu!mania-specific beatmaps.**
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: change line to 
```
The Co-Op mod gives **no** penalty on osu!mania-specific beatmaps.
```

File: wiki/Game_Modifiers/en.md [hunk 5]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
 
-### <img src="Random.png" title="fig:Random.png" alt="Random.png" width="69" height="65" /> Random
+### Random
 
-<img src="GM Random.jpg" title="Effect of &quot;Random&quot; mod" alt="Effect of &quot;Random&quot; mod" width="300" />
+![Random.png](img/Random.png "Random.png")
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use better alt and hover text
`Random mod icon`

File: wiki/Game_Modifiers/en.md [hunk 6]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
+![Effect of "Random" mod](img/GM_Random.jpg "Effect of "Random" mod")
 
 '**'Score multiplier: 1.00 (Unranked, osu!mania only)**'
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use quote?
`> Score multiplier: 1.00 (osu!mania; unranked)`

File: wiki/Game_Modifiers/en.md [hunk 7]
```diff
@@ -146,132 +165,154 @@ Clicking Fade In again will turn on Hidden mod.
...
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: line 319: use quote?
`> Shuffle around the notes!`

File: wiki/Game_Modifiers/en.md [hunk 8]
```diff
@@ -282,14 +323,15 @@ This mod will **change the note placement** only. It does not affect the timelin
 Experimental Mods
 =================
 
-<span style="color:red">**WARNING:**</span> These mods does not appear on "Stable" build. Due to its experimental nature, please proceed at your own risk. Also, the current name is probably a placeholder name. It might change.
+**WARNING:** These mods does not appear on "Stable" build. Due to its experimental nature, please proceed at your own risk. Also, the current name is probably a placeholder name. It might change.
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: change entire line to
```
**Notice:** These mods may not appear in the "Stable" build due to their experimental nature. Please proceed at your own risk!
```

File: wiki/Game_Modifiers/en.md [hunk 9]
```diff
@@ -282,14 +323,15 @@ This mod will **change the note placement** only. It does not affect the timelin
...
 
 Special Mods
 ------------
 
-### <img src="Target.png" title="fig:Target.png" alt="Target.png" width="69" height="65" /> Target Practice
+### Target Practice
 
-<img src="GM_TP.jpg" title="Comparison between original and Target Practice" alt="Comparison between original and Target Practice" width="300" />
+![Target.png](img/Target.png "Target.png")
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: change alt and hover text
`Target Practice mod icon`

File: wiki/Game_Modifiers/en.md [hunk 10]
```diff
@@ -282,14 +323,15 @@ This mod will **change the note placement** only. It does not affect the timelin
...
+![Comparison between original and Target Practice](img/GM_TP.jpg "Comparison between original and Target Practice")
 
 ***Score multiplier: 1.00 (Unranked, osu! only)***
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use quote?
`> Score multiplier: 1.00 (osu!; unranked)`

File: wiki/Game_Modifiers/en.md [hunk 11]
```diff
@@ -282,14 +323,15 @@ This mod will **change the note placement** only. It does not affect the timelin
...
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: line 359:
use this link for multiplayer
`[Multi](wiki/Multi/)`

File: wiki/Game_Modifiers/en.md [hunk 12]
```diff
@@ -323,75 +365,39 @@ These mods were used some time ago, but are now resting here in peace.
 
 ### 10K
 
-<img src="10k.png" title="10k.png" alt="10k.png" width="69" height="65" />
+![10k.png](img/10k.png "10k.png")
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use better alt and hover text
`10K mod icon`

File: wiki/Game_Modifiers/en.md [hunk 13]
```diff
@@ -323,75 +365,39 @@ These mods were used some time ago, but are now resting here in peace.
...
 
 This mod used to enable the player to use 10 keys. (Though, you could still use 5K + COOP to achieve 10 Keys)
 
 ### Fade Out
 
-<img src="FadeOut.png" title="FadeOut.png" alt="FadeOut.png" width="69" height="65" />
+![FadeOut.png](img/FadeOut.png "FadeOut.png")
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use better alt and hover text
`Fade Out mod icon`

File: wiki/Game_Modifiers/en.md [hunk 14]
```diff
@@ -323,75 +365,39 @@ These mods were used some time ago, but are now resting here in peace.
...
 
-This mod was similar to Hidden but was strictly for [Mania](Mania "wikilink"). When enabled, this mod will place a curtain at the bottom of the stage (or top, if upsidedown stage is enabled), fading the notes out when before they reach the keys. This mod's functionality truly wasn't removed, but rather, it was called "Hidden," using the Hidden mod icon.
+This mod was similar to Hidden but was strictly for [Mania](/wiki/Game_Modes/osu!mania). When enabled, this mod will place a curtain at the bottom of the stage (or top, if upsidedown stage is enabled), fading the notes out when before they reach the keys. This mod's functionality truly wasn't removed, but rather, it was called "Hidden," using the Hidden mod icon.
 
 ### No Video
 
-<img src="No Video.png" title="No Video.png" alt="No Video.png" width="69" height="65" />
+![No_Video.png](img/No_Video.png "No_Video.png")
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: use better alt and hover text
`No Video mod icon`

File: wiki/Game_Modifiers/en.md [hunk 15]
```diff
@@ -323,75 +365,39 @@ These mods were used some time ago, but are now resting here in peace.
...
 
-This mod does exactly what you might expect it to do, disable the video, if any. This mod's function was moved to [\#Visual Settings](#Visual_Settings "wikilink") and the icon is no longer being used.
+This mod does exactly what you might expect it to do, disable the video, if any. This mod's function was moved to Visual Settings and the icon is no longer being used.
 
 Visual Settings
 ===============
 
 ### How to find Visual Settings
```
Comment (MegaApplePi) [inline wiki/Game_Modifiers/en.md]: remove this heading

File: wiki/Game_Modifiers/en.md [hunk 16]
```diff
@@ -1,297 +1,347 @@
+# Game modifiers
```
Comment (TPGPL) [inline wiki/Game_Modifiers/en.md]: capitalize the title

File: wiki/Game_Modifiers/en.md [hunk 17]
```diff
@@ -133,7 +133,7 @@ In [osu!catch](/wiki/Game_Modes/osu!catch) mode, the speed and approach rate are
 
 >uguuuuuuuu
 
-If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a bumpin' bass beat to the background. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because **Double Time will automatically be activated in order to use Nightcore mod**.
+If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a drum track to the background that that (in 4/4 time) goes in the order of bass drum on beats 1 & 3, clap on beats 2 & 4, and crash on beat 1 every 4 measures. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because **Double Time will automatically be activated in order to use Nightcore mod**.
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: That second "that" vexes me,  but other than that it looks good.

File: wiki/Game_Modifiers/en.md [hunk 18]
```diff
@@ -1,297 +1,332 @@
...
 
-Note that when multiple mods are applied, the product of the score multipliers is given instead of the sum of them. For example, when Hidden and Flashlight are applied at the same time, they give 1.1872x because (1.06 x 1.12) = 1.1872, rather than (1.06 + 1.12 - 1.00) = 1.18. osu! used to round this number down to the nearest hundredth (0.01), so Hidden and Flashlight together would appear to give a 1.18x multiplier, denying the supposed number round up. Thankfully, a patch corrected this, and now Hidden + Flashlight rounds correctly to 1.19x. <img src="Soloplay8.png" title="fig:osu! List of possible mods" alt="osu! List of possible mods" width="400" />
+![Game mods screen](img/Soloplay8.png "Game mods screen")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Think you missed one.

File: wiki/Game_Modifiers/en.md [hunk 19]
```diff
@@ -1,297 +1,332 @@
...
+In [osu!catch](/wiki/Game_Modes/osu!catch) mode, the effects are the same as in osu! mode - all fruits fall slower (same as reducing AR in osu! mode), the camera is zoomed in (increased circle size in osu! mode) and 3 lives are granted. Despite only using one axis, however, the fruits' larger size and slower falling rate causes them to clump together, creating the same alleged problems as in normal osu!
 
-In [Catch the Beat](Catch_the_Beat "wikilink") mode, the effects are the same as in osu! mode - all fruits fall slower (same as reducing AR in osu! mode), the camera is zoomed in (increased circle size in osu! mode) and 3 lives are granted. Despite only using one axis, however, the fruits' larger size and slower falling rate causes them to clump together, creating the same alleged problems as in normal osu!.
+![Easy decreases the difficulty in many ways.](img/easymod.jpg "Easy decreases the difficulty in many ways.") ![Sometimes, though, Easy just gives you a headache.](img/easymod2.jpg "Sometimes, though, Easy just gives you a headache.")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Here, too.

File: wiki/Game_Modifiers/en.md [hunk 20]
```diff
@@ -1,297 +1,332 @@
...
+In [osu!taiko](/wiki/Game_Modes/osu!taiko) mode, since the gameplay does not end even if the HP bar is zero and only fails the player if their life is too low at the end of the map, this mod allows the player to pass at the end even if the life bar is lower than the required amount to pass.
 
-In [Catch the Beat](Catch_the_Beat "wikilink") mode, this mod acts exactly the same as in standard osu!.
+In [osu!catch](/wiki/Game_Modes/osu!catch) mode, this mod acts exactly the same as in standard osu!
 
-### <img src="Half-time.png" title="fig:Half-time.png" alt="Half-time.png" width="69" height="65" /> Half Time
+### Half Time
 
-<img src="GM_HT.jpg" title="Length of the song is increased by 1/3 BPM decreased by 1/4" alt="Length of the song is increased by 1/3 BPM decreased by 1/4" width="300" />
+![Half Time mod icon](img/Half-time.png "Half Time mod icon")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: As well as here.

File: wiki/Game_Modifiers/en.md [hunk 21]
```diff
@@ -1,297 +1,332 @@
...
 
-*'*Score multiplier: 0.3 (osu!, Taiko, Catch The Beat), 0.5 (osu!mania) *'*
+![Length of the song is increased by 1/3 BPM decreased by 1/4](img/GM_HT.jpg "Length of the song is increased by 1/3 BPM decreased by 1/4")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Here too.

File: wiki/Game_Modifiers/en.md [hunk 22]
```diff
@@ -1,297 +1,332 @@
...
+In [osu!mania](osu!/wiki/Game_Modes/osu!mania) mode, however, Hard Rock will not affect the note placement. It will only raise the judgement difficulty (OD) and the Approach Rate (AR). For changing the note placement, see "Random" mod under Special. Using this mod on osu!mania will only allow unranked play.
 
-<img src="SD Rarecase.jpeg" title="Example of a rare case of surviving Sudden Death. Player managed to grab a slider tick that fills up the HP bar from depleting completely. Results screen" alt="Example of a rare case of surviving Sudden Death. Player managed to grab a slider tick that fills up the HP bar from depleting completely. Results screen" width="200" />
+###  Sudden Death
+![Sudden Death mod icon](/shared/Sudden_Death.png "Sudden Death mod icon")
+![Example of a rare case of surviving Sudden Death. The player managed to grab a slider tick that filled up the HP bar before depleting completely.](img/SD_Rarecase.jpeg "Example of a rare case of surviving Sudden Death. Player managed to grab a slider tick that filled up the HP bar before depleting completely.")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Here as well (SD_Rarecase.jpeg).

File: wiki/Game_Modifiers/en.md [hunk 23]
```diff
@@ -1,297 +1,332 @@
...
 
-In [Catch the Beat](Catch_the_Beat "wikilink") mode, the speed and approach rate are changed the same as in other modes. However, since Catch the Beat has no overall difficulty to increase, this mod does nothing more than a BPM and approach rate multiplier and is thus only worth a 1.06x multiplier. Being the opposite of Half Time, though, Double Time greatly decreases the hyperdash leniency, making fruits requiring hyperdashes nigh impossible to catch in some cases.
+In [osu!catch](/wiki/Game_Modes/osu!catch) mode, the speed and approach rate are changed the same as in other modes. However, since Catch the Beat has no overall difficulty to increase, this mod does nothing more than a BPM and approach rate multiplier and is thus only worth a 1.06x multiplier. Being the opposite of Half Time, though, Double Time greatly decreases the hyperdash leniency, making fruits requiring hyperdashes nigh impossible to catch in some cases.
 
-#### <img src="Nightcore.png" title="fig:Nightcore.png" alt="Nightcore.png" width="69" height="65" /> Nightcore
+#### Nightcore
 
-***Click on Double Time again to activate Nightcore mod***
+![Nightcore mod icon](img/Nightcore.png "Nightcore mod icon")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Missed a spot.

File: wiki/Game_Modifiers/en.md [hunk 24]
```diff
@@ -1,297 +1,332 @@
...
+**Description**: uguuuuuuuu
 
-[This mod was originated from an April Fool joke.](https://osu.ppy.sh/forum/t/49733)
+If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a drum track to the background that (in 4/4 time) goes in the order of bass drum on beats 1 & 3, clap on beats 2 & 4, and crash on beat 1 every 4 measures. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because **Double Time will automatically be activated in order to use Nightcore mod**.
 
-### <img src="Hidden.png" title="fig:Hidden.png" alt="Hidden.png" width="69" height="65" /> Hidden
+### Hidden
 
-***Score multiplier: 1.06*** (osu!,Taiko,CtB) , ***1.00*** (osu!mania)
+![Hidden mod icon](img/Hidden.png "Hidden mod icon")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: I guess you're like an Inkling, splattering haphazardly (jk).

File: wiki/Game_Modifiers/en.md [hunk 25]
```diff
@@ -1,297 +1,332 @@
...
+**Score multiplier**: 1.00 (osu!mania only)
 
 This mod is specially for osu!mania only. It causes notes to fade in as they approach the judgement bar. The speed at which they fade is based on your combo, with higher combos causing the notes to fade in faster and remain invisible for longer.
 
 Clicking Fade In again will turn on Hidden mod.
 
 For a comparison, see pictures below.
 
-<File:GM> o!m FI.jpg|Fade In <File:GM> o!m HD.jpg|Hidden <File:GM> o!m FI(2).jpg|Hidden (DDR style) <File:GM> o!m HD(2).jpg|Fade In (DDR style) <File:Flashlight> Mania.jpg|Flashlight
+![Fade In](img/GM_o!m_FI.jpg "Fade In") ![Hidden](img/GM_o!m_HD.jpg "Hidden") ![Hidden (DDR style)](img/GM_o!m_FI2.jpg "Hidden (DDR style)") ![Fade In (DDR style)](img/GM_o!m_HD2.jpg "Fade In (DDR style)") ![Flashlight](img/Flashlight_Mania.jpg "Flashlight")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Can't forget these.

File: wiki/Game_Modifiers/en.md [hunk 26]
```diff
@@ -1,297 +1,332 @@
...
 
-<File:GM> FI comp.jpg| Fade In combo comparison <File:GM> HD comp.jpg| Hidden combo comparison
+![Fade In combo comparison](img/GM_FI_comp.jpg "Fade In combo comparison") ![Hidden combo comparison](img/GM_HD_comp.jpg "Hidden combo comparison")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Or these.

File: wiki/Game_Modifiers/en.md [hunk 27]
```diff
@@ -1,297 +1,332 @@
...
 These mods introduce new elements to make the game harder and rewards the player in terms of an increase in score multiplier.
 
-### <img src="Hard Rock.png" title="fig:Hard Rock.png" alt="Hard Rock.png" width="69" height="65" /> Hard Rock
-
-<img src="hardrock.jpg" title="Hard Rock increases the difficulty in many ways." alt="Hard Rock increases the difficulty in many ways." width="400" />
+### Hard Rock
 
-***Score multiplier: 1.06 (osu!, Taiko), 1.12 (Catch the Beat), 1.00 (Mania, unranked)***
+![Hard Rock mod icon.png](img/Hard_Rock.png "Hard Rock mod icon.png")
+![Hard Rock increases the difficulty in many ways.](img/hardrock.jpg "Hard Rock increases the difficulty in many ways.")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: As well as here.

File: wiki/Game_Modifiers/en.md [hunk 28]
```diff
@@ -1,297 +1,332 @@
...
+If you click the Sudden Death mod again, the Perfect mod will come out. When this mod is activated, any imperfect hit will be counted as a **miss** and the song will **restart automatically** - something Sudden Death doesn't do. While this mod gives no score multiplier, it is more unforgiving (100% accuracy or auto reset until you give up) when compared to Sudden Death (as long as the combo did not break).
 
-### <img src="Double time.png" title="fig:Double time.png" alt="Double time.png" width="69" height="65" /> Double Time
+### Double Time
 
-<img src="GM_DT.jpg" title="Length of the song is decreased by 1/3 BPM increases by 1/2" alt="Length of the song is decreased by 1/3 BPM increases by 1/2" width="300" />
+![Double_time.png](/shared/Double_time.png "Double_time.png")
 
-***Score multiplier: 1.12 (osu!, Taiko) , 1.06 (Catch the Beat)***
+![Length of the song is decreased by 1/3 BPM increases by 1/2](img/GM_DT.jpg "Length of the song is decreased by 1/3 BPM increases by 1/2")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Can't forget good ol' line 114!

File: wiki/Game_Modifiers/en.md [hunk 29]
```diff
@@ -1,297 +1,332 @@
...
+In [osu!catch](/wiki/Game_Modes/osu!catch), the flashlight is the same, except that it follows the character. The visible area is greater than in standard osu! or in Taiko, but is still quite a challenge as it disables you entirely from seeing notes on the top or the other side of the screen. When paired with Hidden, fruits are visible momentarily if you are directly under them until you reach 100 combo, at which point they become completely invisible.
 
-If you want to know more, refer to these two threads:
+In [osu!mania](osu!/wiki/Game_Modes/osu!mania), vision is restricted to a thin horizontal bar in the center of the screen.
 
--   \[<http://osu.ppy.sh/forum/viewtopic.php?f=9&t=41519>| Flashlight is back!\]
--   \[<http://osu.ppy.sh/forum/viewtopic.php?f=9&t=41039>| Flashlight mod disabled \#2\]
+![Flashlight mod in Standard.](img/flashlightosu.jpg "Flashlight mod in Standard.") ![Flashlight mod in Taiko.](img/flashlighttaiko.jpg "Flashlight mod in Taiko.") ![Flashlight mod in Catch the Beat.](img/flashlightctb.jpg "Flashlight mod in Catch the Beat.") ![Flashlight mod in osu!mania.](img/Flashlight_Mania.jpg "Flashlight mod in osu!mania.")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Don't leave the flashlight in the dark! (Get it?)

File: wiki/Game_Modifiers/en.md [hunk 30]
```diff
@@ -1,297 +1,332 @@
...
 
-#### <img src="Cinema.png" title="fig:Cinema.png" alt="Cinema.png" width="69" height="65" /> Cinema
+In [osu!catch](/wiki/Game_Modes/osu!catch) mode, the Auto mod will actually go beyond perfect and use unlimited speed to catch every fruit in a spinner, despite this being literally impossible for a human player.
 
-<img src="GM_Cinema_comp.jpg" title="Comparison of Auto and Cinema mod" alt="Comparison of Auto and Cinema mod" width="200" />
+#### Cinema
 
-***Click on Auto again to get Cinema mod***
+![Cinema mod icon](/shared/Cinema.png "Cinema mod icon")
+![Comparison of Auto and Cinema mod](img/GM_Cinema_comp.jpg "Comparison of Auto and Cinema mod")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: This review isn't that cinematic, now is it?

File: wiki/Game_Modifiers/en.md [hunk 31]
```diff
@@ -1,297 +1,332 @@
...
+
+
 **The mod does not work on osu!mania-specific beatmaps!**
 
-### <img src="Key_Coop.png" title="fig:Key_Coop.png" alt="Key_Coop.png" width="69" height="65" /> Co-Op
+### Co-Op
 
-<img src="GM COOP2.jpg" title="fig:Comparison between original{Above} and Co-Op{Below} for non-specific difficulty." alt="Comparison between original{Above} and Co-Op{Below} for non-specific difficulty." width="100" /> <img src="GM COOP.jpg" title="fig:Comparison between original{Above} and Co-Op{Below} for osu!mania-specific difficulty." alt="Comparison between original{Above} and Co-Op{Below} for osu!mania-specific difficulty." width="200" />
+![Co-Op mod icon](/shared/Key_Coop.png "Co-Op mod icon")
+![Comparison between original{Above} and Co-Op{Below} for non-specific difficulty.](img/GM_COOP2.jpg "Comparison between original{Above} and Co-Op{Below} for non-specific difficulty.") ![Comparison between original{Above} and Co-Op{Below} for osu!mania-specific difficulty.](img/GM_COOP.jpg "Comparison between original{Above} and Co-Op{Below} for osu!mania-specific difficulty.")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Co-operation is key.

File: wiki/Game_Modifiers/en.md [hunk 32]
```diff
@@ -1,297 +1,332 @@
...
 
-### <img src="Random.png" title="fig:Random.png" alt="Random.png" width="69" height="65" /> Random
+The Co-Op mod gives **no** penalty on osu!mania-specific beatmaps.
 
-<img src="GM Random.jpg" title="Effect of &quot;Random&quot; mod" alt="Effect of &quot;Random&quot; mod" width="300" />
+### Random
 
-'**'Score multiplier: 1.00 (Unranked, osu!mania only)**'
+![Random mod icon](img/Random.png "Random mod icon")
+![Effect of "Random" mod](img/GM_Random.jpg "Effect of "Random" mod")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Awfully random to have missed some of these...

File: wiki/Game_Modifiers/en.md [hunk 33]
```diff
@@ -1,297 +1,332 @@
...
 
 Special Mods
 ------------
 
-### <img src="Target.png" title="fig:Target.png" alt="Target.png" width="69" height="65" /> Target Practice
+### Target Practice
 
-<img src="GM_TP.jpg" title="Comparison between original and Target Practice" alt="Comparison between original and Target Practice" width="300" />
+![Target Practice mod icon](/shared/Target.png "Target Practice mod icon")
+![Comparison between original and Target Practice](img/GM_TP.jpg "Comparison between original and Target Practice")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Instead of targets, practice being thorough with your code, maybe?

File: wiki/Game_Modifiers/en.md [hunk 34]
```diff
@@ -323,75 +358,48 @@ These mods were used some time ago, but are now resting here in peace.
...
-<td align="center"><p>Requires a retry to take effect.</p></td>
-</tr>
-<tr class="odd">
-<td align="center"><p>Disable Video</p></td>
-<td align="center"><p>Doesn't play the video, if any. <span style="text-decoration:underline;">This does not remove the storyboard.</span> This is disabled by default, if there isn't a video to play.</p></td>
-<td align="center"><p>Requires a retry if activated after the song starts.</p></td>
-</tr>
-</tbody>
-</table>
+![Example of Visual Settings](img/VisualSettings.jpg "Example of Visual Settings")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Almost made it! One more.

**Comment by EpicSupermind** [comment]
Only the mod icons should be moved to shared, as they are probably the only images used in other articles.

**Comment by Shadowtail117** [comment]
Alright, I can see that. Got rid of the ones specific to gamemodes, but a couple icons are still in /img/.

File: wiki/Game_Modifiers/en.md (hunk 30 above)
Comment (EpicSupermind) [inline wiki/Game_Modifiers/en.md]: what do you mean it should be moved?

Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Sorry. I just meant that the reference should be switched from img to shared.

File: wiki/Game_Modifiers/en.md [hunk 35]
```diff
@@ -1,297 +1,333 @@
...
+**Score multiplier**: 1.00 (osu!mania only)
 
 This mod is specially for osu!mania only. It causes notes to fade in as they approach the judgement bar. The speed at which they fade is based on your combo, with higher combos causing the notes to fade in faster and remain invisible for longer.
 
 Clicking Fade In again will turn on Hidden mod.
 
 For a comparison, see pictures below.
 
-<File:GM> o!m FI.jpg|Fade In <File:GM> o!m HD.jpg|Hidden <File:GM> o!m FI(2).jpg|Hidden (DDR style) <File:GM> o!m HD(2).jpg|Fade In (DDR style) <File:Flashlight> Mania.jpg|Flashlight
+![Fade In](img/GM_o!m_FI.jpg "Fade In") ![Hidden](img/GM_o!m_HD.jpg "Hidden") ![Hidden (DDR style)](img/GM_o!m_FI2.jpg "Hidden (DDR style)") ![Fade In (DDR style)](img/GM_o!m_HD2.jpg "Fade In (DDR style)") ![Flashlight](img/Flashlight_Mania.jpg "Flashlight")
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: Shouldn't Flashlight_Mania be in the Flashlight section, rather than the Fade In section?

Comment (EpicSupermind) [inline wiki/Game_Modifiers/en.md]: i have no idea why the original wiki did that

File: wiki/Game_Modifiers/en.md [hunk 36]
```diff
@@ -1,297 +1,333 @@
...
+![Nightcore mod icon](/wiki/shared/Nightcore.png "Nightcore mod icon")
 
-***"uguuuuuuuu"***
+Click on Double Time again to activate Nightcore mod
 
-If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a bumpin' bass beat to the background. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because <u>**Double Time will automatically be activated in order to use Nightcore mod**.</u>
+**Description**: uguuuuuuuu
 
-[This mod was originated from an April Fool joke.](https://osu.ppy.sh/forum/t/49733)
+If you click the Double Time icon again, it becomes the Nightcore mod which increases the pitch and adds a drum track to the background that (in 4/4 time) goes in the order of bass drum on beats 1 & 3, clap on beats 2 & 4, and crash on beat 1 every 4 measures. The multiplier stays the same, and the beatmap itself is unaffected beyond what Double Time would do because **Double Time will automatically be activated in order to use Nightcore mod**.
```
Comment (Shadowtail117) [inline wiki/Game_Modifiers/en.md]: You could optionally add that in 6/8 (or 3/4) time, the bass beat goes like this:
- Bass drum on beats 1 and 4
- Clap on beats 3 and 6 
- Crash every 4 measures

**Pull request merged**